app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -1,85 +1,53 @@
-import {
-  Area,
-  Button,
-  ButtonVariant,
-  Col,
-  FormikCheckbox,
-  FormikDropdown,
-  FormikText,
-  IOptionItem,
-  OptionItem,
-  Row,
-  Tab,
-  Tabs,
-} from 'components';
+import { Button, ButtonVariant } from 'components/button';
+import { Col } from 'components/col';
+import { Area, IOptionItem, OptionItem } from 'components/form';
+import { FormikCheckbox, FormikDropdown, FormikText } from 'components/formik';
+import { Row } from 'components/row';
+import { Tab, Tabs } from 'components/tabs';
 import { Formik } from 'formik';
-import { IContentApi, useApiEditor } from 'hooks';
-import React, { useEffect, useState } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { useLookup } from 'store';
+import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useContent, useLookup } from 'store/hooks';
 
 import { PropertiesContentForm } from '.';
+import { defaultFormValues } from './constants';
+import { IContentForm } from './interfaces';
 import { TranscriptContentForm } from './TranscriptContentForm';
-import { formContentToApiContent } from './utils';
+import { toForm, toModel } from './utils';
 
 /**
- * Content Form edit and create form for default view. Path will be appeneded with content id.
+ * Content Form edit and create form for default view. Path will be appended with content id.
  * @returns Edit/Create Form for Content
  */
 export const ContentForm: React.FC = () => {
-  const [active, setActive] = useState('properties');
   const navigate = useNavigate();
-  const {
-    storeMediaTypes,
-    state: { mediaTypes },
-  } = useLookup();
-  const [mediaTypeOptions, setMediatTypeOptions] = useState<IOptionItem[]>([]);
-  const api = useApiEditor({ baseURL: `/api/editor/contents/` });
-  const lookupApi = useApiEditor({ baseURL: '/api' });
-  const path = window.location.pathname.toString();
-  const getContentId = () => {
-    return Number(path.substring(path.lastIndexOf('/') + 1));
-  };
-
-  const [mode] = useState<'edit' | 'create'>(getContentId() === 0 ? 'create' : 'edit');
+  const { id } = useParams();
+  const [{ mediaTypes }] = useLookup();
```

Comment:
All our custom hooks that have state will return an array.  That way we can follow the same design pattern as `React.useState()`